### PR TITLE
new exchange: Deribit

### PR DESCRIPTION
**Pull request by kvdveer**
Closes ccxt/ccxt#334

### CODE REVIEW

File: js/deribit.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        let response = await this.privateGetOrderstate ({ 'orderId': id });
+        return this.parseOrder (response['result']);
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let response = await this.privateGetTradeHistory ({ 'instrument': symbol });
+        return this.parseOrders (response['result']);
```
Comment (kroitor) [inline js/deribit.js]: This line confuses trades with orders... Should be `parseTrades` here...

https://github.com/ccxt/ccxt/wiki/Manual#how-orders-are-related-to-trades

Comment (kroitor) [reply js/deribit.js]: Can you, plz, edit this + `parseTrade` to support for fetchMyTrades? ) Thx!

**Comment by kvdveer** [comment]
I have fixed the order/trade confusion, and copied over your simplified orderbook code.

**Comment by kroitor** [comment]
@kvdveer thanks! Will merge soon!

**Pull request merged**